reflex/components/radix/themes/components/avatar.pyi

```diff
@@ -20,17 +22,29 @@ class Avatar(RadixThemesComponent):
         cls,
         *children,
         variant: Optional[
-            Union[Var[Literal["solid", "soft"]], Literal["solid", "soft"]]
+            Union[reflex.vars.Var[Literal["solid", "soft"]], Literal["solid", "soft"]]
         ] = None,
         size: Optional[
             Union[
-                Var[Literal["1", "2", "3", "4", "5", "6", "7", "8", "9"]],
-                Literal["1", "2", "3", "4", "5", "6", "7", "8", "9"],
+                reflex.vars.Var[
+                    Union[
+                        Literal["1", "2", "3", "4", "5", "6", "7", "8", "9"],
+                        reflex.components.core.breakpoints.Breakpoints[
+                            str, Literal["1", "2", "3", "4", "5", "6", "7", "8", "9"]
+                        ],
+                    ]
+                ],
+                Union[
+                    Literal["1", "2", "3", "4", "5", "6", "7", "8", "9"],
+                    reflex.components.core.breakpoints.Breakpoints[
+                        str, Literal["1", "2", "3", "4", "5", "6", "7", "8", "9"]
+                    ],
+                ],
```

Comment:
not clear why this is generating a separate internal `Union`, since there is already an "outer" `Union` wouldn't the following code work the same:

```suggestion
                Literal["1", "2", "3", "4", "5", "6", "7", "8", "9"],
                reflex.components.core.breakpoints.Breakpoints[
                    str, Literal["1", "2", "3", "4", "5", "6", "7", "8", "9"]
                ],
```

Reply:
That's interesting, I'm not sure how it generated this piece of code. I will take a look into it.